src/polar/Pie.tsx

```diff
@@ -137,6 +137,7 @@ interface PieProps extends PieDef {
   className?: string;
   dataKey: DataKey<any>;
   nameKey?: DataKey<any>;
+  radiusKey?: DataKey<any>;
```

Comment:
I also have a concern with the naming of this. There is currently both innerRadius and outerRadius props. Calling the new prop radiusKey might be confusing as to which radius we are affecting? And if you can do it to outerRadius why not inner as well?

Maybe instead of a new prop at all we should just let inner and outer radius take functions which give your data as an argument. Then you can do something like 

outerRadius={(data) => data.someRadius}

That seems a lot simpler